histomicstk/features/compute_global_graph_features.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import division
+
+from collections import namedtuple
+from itertools import combinations
+
+import numpy as np
+from numpy import linalg
+from scipy.spatial import cKDTree as KDTree, Voronoi
+from scipy import sparse
+from scipy.sparse.csgraph import minimum_spanning_tree
+
+PopStats = namedtuple('PopStats', ['mean', 'stddev', 'minmaxr', 'disorder'])
+PolyProps = namedtuple('PolyProps', ['area', 'peri', 'max_dist'])
+TriProps = namedtuple('TriProps', ['sides', 'area'])
+DensityProps = namedtuple('DensityProps', ['neighbors_in_distance',
+                                           'distance_for_neighbors'])
+Props = namedtuple('Props', ['voronoi', 'delaunay', 'mst_branches', 'density'])
+
+
+def compute_global_graph_features(centroids, scale=10., neighbor_counts=(3, 5, 7)):
```

Comment:
If we are considering adding "local" versions of these -- ones that give per-nucleus values -- as well as "global" ones that give values describing an entire region, how about `compute_global_cell_graph_features`?